Initial upload+layout

OmriS1996 opened the pull request:
Created base up with a side menu taken from: https://github.com/trananhtuat/animated-sidebar-indicator

OmriS1996 committed:
Initial upload+layout

Created base up with a side menu taken from: https://github.com/trananhtuat/animated-sidebar-indicator

sshason commented on package.json:
```diff
@@ -0,0 +1,41 @@
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^13.5.0",
```
it seems those can be moved to the devDependencies:

@testing-library/jest-dom": "^5.16.2",
    "@testing-library/react": "^12.1.3",
    "@testing-library/user-event": "^13.5.0",

sshason commented:
Nice!

sshason reviewed (approved):
Nice work!

OmriS1996 committed:
moc data modifications

added moc data and modified files to better acommodate future data

OmriS1996 committed:
Items Start

added an item collection page using MaterialUI, updated mocdaata, api file and files that used it

OmriS1996 committed:
backup

backing up a bit of progress, formating pc
